helm/prometheus-rules/templates/alloy-rules-configmap.yaml

```diff
@@ -45,6 +45,24 @@ data:
             mimir.rules.kubernetes "local" {
               address = "http://mimir-ruler.mimir.svc:8080/"
               tenant_id = "anonymous"
+              rule_selector {
```

Comment:
Maybe we should have alloy in a different namespace than mimir in another PR?

Reply:
I agree, alloy probably belongs to `monitoring` namespace.